### PR TITLE
Aerospike vector search client 0.6.0 review changes

### PR DESCRIPTION
**Pull request by DomPeliniAerospike**

### CODE REVIEW

File: src/aerospike_vector_search/aio/client.py [hunk 1]
```diff
@@ -326,15 +326,15 @@ async def vector_search(
...
+            raise types.AVSServerError(rpc_error=e)
         async_results = []
         async for result in results_stream:
             async_results.append(self._respond_neighbor(result))
 
         return async_results
 
     async def wait_for_index_completion(
-        self, *, namespace: str, name: str, timeout: Optional[int] = sys.maxsize, wait_interval: Optional[int] = 12
+        self, *, namespace: str, name: str, timeout: Optional[int] = sys.maxsize, wait_interval: Optional[int] = 12, validation_threshold: Optional[int] = 2
```
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply src/aerospike_vector_search/aio/client.py:337]: Changed to validation_checks and consecutive_index_validations.

File: src/aerospike_vector_search/client.py [hunk 2]
```diff
@@ -330,7 +331,7 @@ def wait_for_index_completion(
             the timeout is reached or the index has no pending index update operations.
         """
         # Wait interval between polling
-        (index_stub, wait_interval, start_time, unmerged_record_initialized, double_check, index_completion_request) = self._prepare_wait_for_index_waiting(namespace, name, wait_interval)
+        (index_stub, wait_interval, start_time, unmerged_record_initialized, validation_count, index_completion_request) = self._prepare_wait_for_index_waiting(namespace, name, wait_interval)
```
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply src/aerospike_vector_search/client.py]: Changed this to consecutive_index_validations.

[the remark this answers is not in the document]
Comment (jdogmcsteezy) [reply src/aerospike_vector_search/client.py]: hmmmm

**Review by jdogmcsteezy — APPROVED** [review]
LGTM, unless you want to merge the .flake8 file into pyproject.toml

**Pull request merged**